backend/qemu.pm

```diff
@@ -866,6 +866,9 @@ sub start_qemu ($self) {
         }
     }
 
+    my $v6checkhost = $vars->{IPV6CHECKHOST} // 'opensuse.org';
+    $vars->{IPV6AVAILABLE} = !runcmd('ping', '-6', '-c', 1, '-w', 1, '-W', 1, $v6checkhost);
```

Comment:
That might end up as a boolean `true` or `false` in the vars.json because in perl versions >= 5.38 that `!` will automatically convert the result to a boolean (in older versions booleans kind of existed, but were not preserved, so a JSON module would only see a 1 or '').
So this might be fine, I just wanted to add this, because the behaviour would change between current Tumbleweed and Leap 15.6.

Reply:
I've adjusted it to use a ternary operator which makes it a little more explicit and IMHO even more readable.

Reply:
I actually liked it, very perlish :)
Just wanted to mention it